### PR TITLE
Updated Validated Hardware blurb for clarity

### PR DESCRIPTION
**Pull request by jaradtke-aws**
*Issue #, if available:*
The Validated Hardware section was causing confusion, as it implied that there is/was a finite list of systems approved for use with EKS Anywhere.

*Description of changes:*
Updated verbiage to indicate the compatibility ultimately lies with the operating system used to build the images. 
Provided links to customize image/hookOS, if necessary.

*Testing (if applicable):*
N/A

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/getting-started/baremetal/bare-prereq.md [hunk 1]
```diff
@@ -81,8 +81,9 @@ Here are other network requirements:
 
 ## Validated hardware
 
-Through extensive testing in a variety of on-premises environments, we expect Amazon EKS Anywhere on bare metal to run on most generic hardware that meets the above requirements.
-In addition, we have collaborated with our hardware original equipment manufacturer (OEM) partners to provide you a list of validated hardware:
+Through extensive testing in a variety of on-premises environments, we expect Amazon EKS Anywhere on bare metal to run on most generic hardware that meets the above requirements.  Compatibility is determined by the host operating system selected when [Building Node Images](https://quip-amazon.com/6eLsAjMvL5NI/EKS-Anywhere-Validated-Hardware#temp:C:FZU2fe2764a2d3440bea304af090).  Installation may require you to [Customize HookOS for EKS Anywhere on Bare Metal](https://anywhere.eks.amazonaws.com/docs/getting-started/baremetal/customize/bare-custom-hookos/) to add drivers, or modify configuration specific to your environment.  [Bottlerocket support for bare metal was deprecated with the EKS Anywhere v0.19 release.](https://github.com/aws/eks-anywhere/issues/7754)
```
Comment (csplinter) [inline docs/content/en/docs/getting-started/baremetal/bare-prereq.md:84]: You have a link to a Quip doc

File: docs/content/en/docs/getting-started/baremetal/bare-prereq.md [hunk 2]
```diff
@@ -81,8 +81,9 @@ Here are other network requirements:
...
+
+The following is a list from collaboration with our hardware original equipment manufacturer (OEM) partners from previous validation testing.  (this list of examples will no longer be maintained)
```
Comment (csplinter) [inline docs/content/en/docs/getting-started/baremetal/bare-prereq.md:86]: Remove this table entirely. It is doing more harm than good.

File: docs/content/en/docs/getting-started/baremetal/bare-prereq.md (hunk 1 above)
Comment (jaradtke-aws) [reply docs/content/en/docs/getting-started/baremetal/bare-prereq.md]: Done - updated to point to correct link

File: docs/content/en/docs/getting-started/baremetal/bare-prereq.md (hunk 2 above)
Comment (jaradtke-aws) [reply docs/content/en/docs/getting-started/baremetal/bare-prereq.md]: Done - good suggestion.

File: docs/content/en/docs/getting-started/baremetal/bare-prereq.md [hunk 3]
```diff
@@ -81,18 +81,4 @@ Here are other network requirements:
...
-| HPE ProLiant DL20 | iLO5 | HPE 361i 1G |
-| HPE ProLiant DL160 Gen10 | iLO5 | HPE Eth 10/25Gb 2P 640SFP28 A |
-| Dell PowerEdge R340 | iDRAC9 | Broadcom 57416 Dual Port 10GbE |
-| HPE ProLiant DL360 | iLO5 | HPE Ethernet 1Gb 4-port 331i |
-| Lenovo ThinkSystem SR650 V2 | XClarity Controller Enterprise v7.92 | Intel I350 1GbE RJ45 4-port OCP & Marvell QL41232 10/25GbE SFP28<br>2-Port PCIe Ethernet Adapter |
+Through extensive testing in a variety of on-premises environments, we expect Amazon EKS Anywhere on bare metal to run on most generic hardware that meets the above requirements.  Compatibility is determined by the host operating system selected when [Building Node Images](https://anywhere.eks.amazonaws.com/docs/osmgmt/artifacts/#building-node-images).  Installation may require you to [Customize HookOS for EKS Anywhere on Bare Metal](https://anywhere.eks.amazonaws.com/docs/getting-started/baremetal/customize/bare-custom-hookos/) to add drivers, or modify configuration specific to your environment.  [Bottlerocket support for bare metal was deprecated with the EKS Anywhere v0.19 release.](https://github.com/aws/eks-anywhere/issues/7754)
```
Comment (csplinter) [inline docs/content/en/docs/getting-started/baremetal/bare-prereq.md:84]: One more slight tweak, "Through extensive testing in a variety of on-premises environments, we have validated Amazon EKS Anywhere on bare metal works without modification on most modern hardware that meets the above requirements."

**Review by csplinter — APPROVED** [review]

**Pull request merged**